src/globus_sdk/login_flows/__init__.py

```diff
@@ -6,10 +6,10 @@
 )
 from .login_flow_manager import LoginFlowManager
 
-__all__ = [
+__all__ = (
     "CommandLineLoginFlowManager",
     "LocalServerLoginError",
     "LocalServerEnvironmentalLoginError",
     "LocalServerLoginFlowManager",
     "LoginFlowManager",
-]
+)
```

Comment:
Remind me to add a lint for this. IMO `__all__` should almost always be a tuple, not a list.